### PR TITLE
822 remove toggleablebox

### PR DESCRIPTION
**Pull request by StaehliJ**
# Pull request

## Description

The goal of this PR is to remove helpers that are so simple to do that we remove them from the Pillarbox and moved into the demo.

## Changes made

- Remove composable `ToggleableBox`.
- Remove composable `DelayedVisibility`.

## Checklist

- [ ] APIs have been properly documented (if relevant).
- [ ] The documentation has been updated (if relevant).
- [ ] New unit tests have been written (if relevant).
- [ ] The demo has been updated (if relevant).

### CODE REVIEW

File: pillarbox-demo/src/main/java/ch/srgssr/pillarbox/demo/ui/player/PlayerView.kt [hunk 1]
```diff
@@ -156,19 +154,88 @@ fun PlayerView(
...
-            visible = !visibilityState.isVisible
+            interactionSource = interactionSource,
+            progressTracker = progressTracker,
+            credit = currentCredit,
+            content = content
         )
     }
 }
+
+private fun Modifier.onEnterPressed(action: () -> Unit): Modifier {
```
Comment (MGaetan89) [inline pillarbox-demo/src/main/java/ch/srgssr/pillarbox/demo/ui/player/PlayerView.kt:229]: Why not use the existing `onDpadEvent`?

File: pillarbox-demo/src/main/java/ch/srgssr/pillarbox/demo/ui/player/PlayerView.kt [hunk 2]
```diff
@@ -156,19 +154,88 @@ fun PlayerView(
             )
         }
 
-        BlockedTimeRangeWarning(
```
Comment (MGaetan89) [inline pillarbox-demo/src/main/java/ch/srgssr/pillarbox/demo/ui/player/PlayerView.kt:159]: We don't display block messages anymore?

File: pillarbox-demo-shared/src/main/java/ch/srgssr/pillarbox/demo/shared/ui/TalkBackEnabled.kt
[the remark this answers is not in the document]
Comment (StaehliJ) [reply pillarbox-demo-shared/src/main/java/ch/srgssr/pillarbox/demo/shared/ui/TalkBackEnabled.kt:1]: Not exactly, the one in TalkBack checks TouchAccessibilityChanges and this one checks if it is enabled or not.

File: pillarbox-demo/src/main/java/ch/srgssr/pillarbox/demo/ui/player/PlayerView.kt (hunk 2 above)
Comment (StaehliJ) [reply pillarbox-demo/src/main/java/ch/srgssr/pillarbox/demo/ui/player/PlayerView.kt:159]: Yes to be aligned with apple demo :D

File: pillarbox-demo/src/main/java/ch/srgssr/pillarbox/demo/ui/player/PlayerView.kt (hunk 1 above)
Comment (StaehliJ) [reply pillarbox-demo/src/main/java/ch/srgssr/pillarbox/demo/ui/player/PlayerView.kt:229]: Indeed

**Review by MGaetan89 — APPROVED** [review]

**Review by MGaetan89 — APPROVED** [review]

**Pull request merged**